### PR TITLE
Added Commissioner 1.000

### PR DESCRIPTION
**Pull request by yanone**
Upstream PR: https://github.com/kosbarts/Commissioner/pull/34

### CODE REVIEW

File: ofl/commissioner/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  copyright: "Copyright 2019 The Commissioner Project Authors (https://github.com/kosbarts/Commissioner)"
+}
+subsets: "cyrillic"
+subsets: "cyrillic-ext"
+subsets: "greek"
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+subsets: "vietnamese"
+axes {
```
Comment (m4rc1e) [inline ofl/commissioner/METADATA.pb:22]: @rsheeter Since these axes are unregistered, shall we remove them and only keep the wght axis?

Comment (m4rc1e) [reply ofl/commissioner/METADATA.pb:22]: I see Nate recently PR'd the axisregistry. Should we make sure that our METADATA.pb files don't include axes which are not part of this doc?

**Comment by m4rc1e** [comment]
## Fontbakery report

Fontbakery version: 0.7.29

<details>
<summary><b>[1] Family checks</b></summary>
<details>
<summary>⚠ <b>WARN:</b> Is the command `ftxvalidator` (Apple Font Tool Suite) available?</summary>

* [com.google.fonts/check/ftxvalidator_is_available](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/universal.html#com.google.fonts/check/ftxvalidator_is_available)
<pre>--- Rationale ---

There&#x27;s no reasonable (and legal) way to run the command `ftxvalidator` of the
Apple Font Tool Suite on a non-macOS machine. I.e. on GNU+Linux or Windows etc.

If Font Bakery is not running on an OSX machine, the machine running Font
Bakery could access `ftxvalidator` on OSX, e.g. via ssh or a remote procedure
call (rpc).

There&#x27;s an ssh example implementation at:
https://github.com/googlefonts/fontbakery/blob/master/prebuilt/workarounds
/ftxvalidator/ssh-implementation/ftxvalidator


</pre>

* ⚠ **WARN** Could not find ftxvalidator.

</details>
<br>
</details>
<details>
<summary><b>[8] Commissioner[FLAR,VOLM,slnt,wght].ttf</b></summary>
<details>
<summary>💔 <b>ERROR:</b> Check correctness of STAT table strings </summary>

* [com.google.fonts/check/STAT_strings](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/universal.html#com.google.fonts/check/STAT_strings)
<pre>--- Rationale ---

On the STAT table, the &quot;Italic&quot; keyword must not be used on AxisValues for
variation axes other than &#x27;ital&#x27;.


</pre>

* 💔 **ERROR** Failed with AttributeError: AxisIndex

</details>
<details>
<summary>🔥 <b>FAIL:</b> Ensure VFs do not contain slnt or ital axes. </summary>

* [com.google.fonts/check/varfont/unsupported_axes](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/varfont/unsupported_axes)
<pre>--- Rationale ---

The &#x27;ital&#x27; axis is not supported yet in Google Chrome. The &#x27;opsz&#x27; axis also has
patchy support.

For the time being, we need to ensure that VFs do not contain either of these
axes. Once browser support is better, we can deprecate this check.

For more info regarding ital and opsz browser support, see:
https://arrowtype.github.io/vf-slnt-test/


</pre>

* 🔥 **FAIL** The "slnt" axis is not yet well supported on Google Chrome. [code: unsupported-slnt]

</details>
<details>
<summary>⚠ <b>WARN:</b> DESCRIPTION.en_us.html should end in a linebreak.</summary>

* [com.google.fonts/check/description/eof_linebreak](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/description/eof_linebreak)
<pre>--- Rationale ---

Some older text-handling tools sometimes misbehave if the last line of data in
a text file is not terminated with a newline character (also known as &#x27;\n&#x27;).

We know that this is a very small detail, but for the sake of keeping all
DESCRIPTION.en_us.html files uniformly formatted throughout the GFonts
collection, we chose to adopt the practice of placing this final linebreak char
on them.


</pre>

* ⚠ **WARN** The last characther on DESCRIPTION.en_us.html is not a line-break. Please add it. [code: missing-eof-linebreak]

</details>
<details>
<summary>⚠ <b>WARN:</b> Check copyright namerecords match license file.</summary>

* [com.google.fonts/check/name/license](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/license)
<pre>--- Rationale ---

A known licensing description must be provided in the NameID 14 (LICENSE
DESCRIPTION) entries of the name table.

The source of truth for this check (to determine which license is in use) is a
file placed side-by-side to your font project including the licensing terms.

Depending on the chosen license, one of the following string snippets is
expected to be found on the NameID 13 (LICENSE DESCRIPTION) entries of the name
table:
- &quot;This Font Software is licensed under the SIL Open Font License, Version 1.1.
This license is available with a FAQ at: https://scripts.sil.org/OFL&quot;
- &quot;Licensed under the Apache License, Version 2.0&quot;
- &quot;Licensed under the Ubuntu Font Licence 1.0.&quot;


Currently accepted licenses are Apache or Open Font License.
For a small set of legacy families the Ubuntu Font License may be acceptable as
well.

When in doubt, please choose OFL for new font projects.


</pre>

* ⚠ **WARN** Please consider using HTTPS URLs at name table entry [plat=3, enc=1, name=13] [code: http-in-description]
* ⚠ **WARN** For now we're still accepting http URLs, but you should consider using https instead.
 [code: http]

</details>
<details>
<summary>⚠ <b>WARN:</b> License URL matches License text on name table?</summary>

* [com.google.fonts/check/name/license_url](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/name/license_url)
<pre>--- Rationale ---

A known license URL must be provided in the NameID 14 (LICENSE INFO URL) entry
of the name table.

The source of truth for this check is the licensing text found on the NameID 13
entry (LICENSE DESCRIPTION).

The string snippets used for detecting licensing terms are:
- &quot;This Font Software is licensed under the SIL Open Font License, Version 1.1.
This license is available with a FAQ at: https://scripts.sil.org/OFL&quot;
- &quot;Licensed under the Apache License, Version 2.0&quot;
- &quot;Licensed under the Ubuntu Font Licence 1.0.&quot;


Currently accepted licenses are Apache or Open Font License.
For a small set of legacy families the Ubuntu Font License may be acceptable as
well.

When in doubt, please choose OFL for new font projects.


</pre>

* ⚠ **WARN** Please consider using HTTPS URLs at name table entry [plat=3, enc=1, name=13] [code: http-in-description]
* ⚠ **WARN** Please consider using HTTPS URLs at name table entry [plat=3, enc=1, name=13] [code: http-in-description]
* ⚠ **WARN** Please consider using HTTPS URLs at name table entry [plat=3, enc=1, name=13] [code: http-in-description]
* ⚠ **WARN** Please consider using HTTPS URLs at name table entry [plat=3, enc=1, name=14] [code: http-in-license-info]
* ⚠ **WARN** For now we're still accepting http URLs, but you should consider using https instead.
 [code: http]

</details>
<details>
<summary>⚠ <b>WARN:</b> METADATA.pb: Fontfamily is listed on Google Fonts API?</summary>

* [com.google.fonts/check/metadata/listed_on_gfonts](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/metadata/listed_on_gfonts)

* ⚠ **WARN** Family not found via Google Fonts API. [code: not-found]

</details>
<details>
<summary>⚠ <b>WARN:</b> Are there caret positions declared for every ligature?</summary>

* [com.google.fonts/check/ligature_carets](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/ligature_carets)
<pre>--- Rationale ---

All ligatures in a font must have corresponding caret (text cursor) positions
defined in the GDEF table, otherwhise, users may experience issues with caret
rendering.

If using GlyphsApp, ligature carets can be set directly on canvas by accessing
the `Glyph -&gt; Set Anchors` menu option or by pressing the `Cmd+U` keyboard
shortcut.


</pre>

* ⚠ **WARN** This font lacks caret position values for ligature glyphs on its GDEF table. [code: lacks-caret-pos]

</details>
<details>
<summary>⚠ <b>WARN:</b> Is there kerning info for non-ligated sequences?</summary>

* [com.google.fonts/check/kerning_for_non_ligated_sequences](https://font-bakery.readthedocs.io/en/latest/fontbakery/profiles/googlefonts.html#com.google.fonts/check/kerning_for_non_ligated_sequences)
<pre>--- Rationale ---

Fonts with ligatures should have kerning on the corresponding non-ligated
sequences for text where ligatures aren&#x27;t used (eg
https://github.com/impallari/Raleway/issues/14).


</pre>

* ⚠ **WARN** GPOS table lacks kerning info for the following non-ligated sequences:
	- f + f
	- f + i
	- i + f
	- f + l
	- l + f
	- i + l

   [code: lacks-kern-info]

</details>
<br>
</details>

### Summary

| 💔 ERROR | 🔥 FAIL | ⚠ WARN | 💤 SKIP | ℹ INFO | 🍞 PASS | 🔎 DEBUG |
|:-----:|:----:|:----:|:----:|:----:|:----:|:----:|
| 1 | 1 | 7 | 35 | 9 | 124 | 0 |
| 1% | 1% | 4% | 20% | 5% | 70% | 0% |

**Note:** The following loglevels were omitted in this report:
* **SKIP**
* **INFO**
* **PASS**
* **DEBUG**


## Diff images: [commisioner_qa.zip](http://35.238.63.0/media/fb941bec-5903-48a3-a1e3-499e60edc6aa/commisioner_qa.zip)

**Comment by yanone** [comment]
I tried to dig into the ERROR, but no success. A non-existing `AxisValue` attribute is being reported, when it really seems to exist. I aborted that attempt.

**Comment by graphicore** [comment]
> I tried to dig into the ERROR, but no success. A non-existing AxisValue attribute is being reported, when it really seems to exist. I aborted that attempt.

If it's an ERROR it must be fixed in Font Bakery and the problem is a not existing attribute:

> `Failed with AttributeError: AxisIndex`

I suggest to file an issue at https://github.com/googlefonts/fontbakery/issues

**Comment by yanone** [comment]
Okay, will do it

**Comment by m4rc1e** [comment]
The i's sidebearings seem a bit off in the darker weights.

Screenshot 2020-07-22 at 10 32 54

**Comment by yanone** [comment]
I can't confirm that from looking at the sources. I shall try without hinting. How did you get to this preview image?

**Comment by RosaWagner** [comment]
@m4rc1e I run a browser previews on the statics and I couldn't replicate this spacing issue. If it concerns de variable, it should be already unhinted. In any case, such weight shouldn't be use under 30pt… what do you think ?

**Comment by davelab6** [comment]
> such weight shouldn't be use under 30pt… what do you think ?

I think users will still use it at 18 and notice XD I'd like to get to the bottom of why this happens as Yanone saiid he couldn't see it in the source files... 

@kosbarts any suggestions? :)

**Comment by davelab6** [comment]
@m4rc1e I need to get the axis registry updated with the custom axes, too. Marc are you able to make a separate PR for that?

**Comment by kosbarts** [comment]
I am out right now. I will check it again once back in a few hours and respond. On Sep 3, 2020 17:13, Dave Crossland <notifications@github.com> wrote:



such weight shouldn't be use under 30pt… what do you think ?



I think users will still use it at 18 and notice XD I'd like to get to the bottom of why this happens as Yanone saiid he couldn't see it in the source files...

@kosbarts any suggestions? :)

**Comment by RosaWagner** [comment]
So I re-run a browser previews on the variable. The bug appears in Mac OS High Sierra Safari preview:

Capture d’écran 2020-09-04 à 11 19 11

But not in Windows 10 Chrome:

Capture d’écran 2020-09-04 à 11 18 43

When I play in Safari (Mas OS Mojave), the spacing do have a weird behaviour, which in not the case in Chrome:
[video-commissioner.zip](https://github.com/google/fonts/files/5173931/video-commissioner.zip)

**Comment by RosaWagner** [comment]
So, this spacing issue is apparently a bug of Safari with all composites. It is fixed in Catalina apparently. We could satisfy the diffbrowser preview by decomposing the /i. Although it bothers a lot the designer because it would require a special script in his production process. And what about the other composites? He would prefer to let it like that, or to decompose everything (but 16 masters in multiscripts would make the the file heavier).

**Comment by davelab6** [comment]
I'm OK to proceed

**Comment by RosaWagner** [comment]
Made a new PR https://github.com/google/fonts/pull/2675 with regenerated fonts and added informations.

**Pull request closed without merging**